django/core/checks/urls.py

```diff
@@ -44,9 +46,18 @@ def get_warning_for_invalid_pattern(pattern):
     describe_pattern() cannot be used here, because we cannot rely on the
     urlpattern having regex or name attributes.
     """
+    if isinstance(pattern, six.string_types):
+        hint = "Try removing the string {!r}. The list of urlpatterns should " \
```

Comment:
Typically we've shied away from using `__repr__` for strings in messages because this may output the `u''` prefix on Python 2 which is a bit unsightly. I don't have a strong opinion about it, just mentioning off hand.

Also we usually use this style to avoid backslashes and non-4 space indentation.
```
hint = (
    '....'
)